### PR TITLE
3652 link maps to cache

### PR DESCRIPTION
**Pull request by trystant**
In your PR did you:

  - [X] Include a description of the changes? Implemented HERE Maps marker cluster getting information from Redis
  - [X] Mention the issue the PR addresses? #3652 
  - [X] Include screenshots of any changes to the UI?
Screenshot 2020-09-21 at 03 50 33 - Display 2
  - [ ] Isolate any changes to gems (meaning that any new, updated, or removed gems and resulting code changes should be in their own PR)?
  - [X] Add and/or update specs for your code?

### CODE REVIEW

**Comment by rlgreen91** [comment]
@trystant Got some conflicts

**Comment by trystant** [comment]
I'm running into the following error when pulling up `https:/staging.ebwiki.org/maps`. Here's the [rollbar](https://rollbar.com/trystant/EBWiki/items/2450/occurrences/156947916567/) link.

**Comment by rlgreen91** [comment]
@trystant merge conflicts

**Comment by mnyon-grandkru** [comment]
@rlgreen91 the conflicts are resolved.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -33,13 +33,13 @@
...
     </script>
-
     <script type="text/javascript" src="https://js.api.here.com/v3/3.1/mapsjs-core.js"></script>
     <script type="text/javascript" src="https://js.api.here.com/v3/3.1/mapsjs-service.js"></script>
+    <script type="text/javascript" src="https://js.api.here.com/v3/3.1/mapsjs-clustering.js"></script>
     <script type="text/javascript" src="https://js.api.here.com/v3/3.1/mapsjs-ui.js"></script>
     <script type="text/javascript" src="https://js.api.here.com/v3/3.1/mapsjs-mapevents.js"></script>
     <script type="text/javascript" >window.ENV_VARIABLE = 'https://<%= request.host %>'</script>
-    <script src='https://developer.here.com/javascript/src/iframeheight.js'></script>
+    <script type="text/javascript" src='https://developer.here.com/javascript/src/iframeheight.js'></script>
```
Comment (rlgreen91) [inline app/views/layouts/application.html.erb:42]: NIT: Is it possible to use `async` or `defer` here to improve loading?

File: app/views/maps/index.html.erb [hunk 2]
```diff
@@ -15,10 +14,10 @@
 			</div>
 			<h2 class="page-subheader">Click on the map pins below to learn more.</h2>
 			<div id="markers-on-the-map">
-			    <p>This example displays a movable map initially centered on the <b>United States</b> in the centre of Berlin <i>(52.5159°N, 13.3777°E)</i></p>
 	    		<div id="map page-header">
-		    		<div style="width: 640px; height: 480px; margin: auto" id="mapContainer"></div>
-
+		    		<div style="width: 960px; height: 720px; margin: auto" id="mapContainer"></div>
```
Comment (rlgreen91) [inline app/views/maps/index.html.erb:18]: Please put this in a `maps.scss` file.

File: spec/helpers/maps_helper_spec.rb [hunk 3]
```diff
@@ -4,10 +4,9 @@
 
 RSpec.describe MapsHelper, type: :helper do
   describe '#fetch_cases' do
-    before :each do
-      $redis.set('cases', '')
-    end
     it 'fetches cases from the database if they do not exist in the cache' do
+      pending
```
Comment (rlgreen91) [inline spec/helpers/maps_helper_spec.rb:8]: Why is this pending?

**Review by rlgreen91 — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (mnyon-grandkru) [reply app/views/layouts/application.html.erb]: I can add those & compare script loading times.

File: app/views/maps/index.html.erb (hunk 2 above)
Comment (mnyon-grandkru) [reply app/views/maps/index.html.erb:18]: Yup. I'll adjust that.

**Comment by mnyon-grandkru** [comment]
@rlgreen91 @managedkaos It looks like there are some Github Action failures on some checks due to not being able to resolve Ubuntu packages needed. In particular, this error message stood out:
`Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/c/curl/libcurl4-openssl-dev_7.68.0-1ubuntu2.4_amd64.deb`
Looking at the libraries, it seems that `http://security.ubuntu.com/ubuntu/pool/main/c/curl/curl_7.68.0-1ubuntu2.5_amd64.deb` is available, but 2.4 is what's being searched. I'll look into how this can be updated.

**Comment by managedkaos** [comment]
Let me check it. May need to update the OS version. Also, if these builds are failing checks, then it's the action image not our image.

**Review by rlgreen91 — APPROVED** [review]
LGTM

**Pull request merged**